### DISCUSSION TITLE
Newsletter (Weekly or Daily)

### DISCUSSION THREAD

**Discussion by lawgimenez** [community/community#2893] [Discussions] [answered]
I will be able to subscribe to a discussion newsletter of any repository of my choosing so I can read it on weekends.

**Comment by becca** [comment, marked as answer]
Thank you for submitting this feedback! We are tracking a request similar to this across other features on GitHub. cc// @benjam

**Comment by brinafloatzzzzz** [comment]
newsletters//sendatemailz//.......

**Comment by brinafloatzzzzz** [comment]
sendemail.
